### PR TITLE
New Module: na_elementsw_initiators

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
New Module: na_elementsw_initiators.py
##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
- na_elementsw_initiators.py

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Review by lonico — APPROVED** [review]
shipit

**Review by thedoubl3j — APPROVED** [review]
followed CRUD and simple in operation. CI and tests are good, lgtm.

File: lib/ansible/modules/storage/netapp/na_elementsw_initiators.py [hunk 1]
```diff
@@ -0,0 +1,317 @@
+#!/usr/bin/python
+# (c) 2018, NetApp, Inc
```
Comment (thedoubl3j) [inline lib/ansible/modules/storage/netapp/na_elementsw_initiators.py:2]: assuming you guys might want to add 2018-2019 @carchi8py ?

Comment (carchi8py) [reply lib/ansible/modules/storage/netapp/na_elementsw_initiators.py:2]: Let go ahead and merge it as is, and we can come back and change that to 2019 later.

Comment (thedoubl3j) [reply lib/ansible/modules/storage/netapp/na_elementsw_initiators.py:2]: 👍

**Comment by thedoubl3j** [comment]
cc @thaumos

**Pull request merged**